build.sbt

```diff
@@ -100,12 +101,43 @@ lazy val remote = project.dependsOn(coreJVM).settings(
   libraryDependencies ++= jvmDependencies
 )
 
-lazy val vision = project.dependsOn(coreJVM).settings(
+lazy val vision = crossProject(JVMPlatform, NativePlatform).crossType(CrossType.Pure).dependsOn(core).settings(
   name := "potassium-vision",
-  libraryDependencies ++= sharedDependencies.value,
-  libraryDependencies ++= jvmDependencies
+  libraryDependencies ++= sharedDependencies.value
+).jvmSettings(
+  libraryDependencies ++= jvmDependencies,
+  resolvers += "WPILib-Maven" at "http://team846.github.io/wpilib-maven",
+  resolvers += "opencv-maven" at "https://github.com/WPIRoboticsProjects/opencv-maven/raw/mvn-repo",
+
+  libraryDependencies += "edu.wpi.first" % "cscore" % "2018.1.1",
+  libraryDependencies += "org.opencv" % "opencv-java" % "3.1.0",
+  libraryDependencies += "edu.wpi.first" % "ntcore" % wpiVersion
+).nativeSettings(
+  if (System.getenv("NATIVE_TARGET") == "ARM32") {
+    Seq(
+      libraryDependencies += "com.lynbrookrobotics" %%% "wpilib-scala-native" % "0.1-SNAPSHOT",
+      libraryDependencies += "edu.wpi.first" % "cscore" % "2018.1.1",
+      libraryDependencies += "org.opencv" % "opencv-java" % "3.1.0",
+      libraryDependencies += "com.lynbrookrobotics" %%% "ntcore-scala-native" % "0.1.0+4-f5af12c9",
+      libraryDependencies += "com.lynbrookrobotics" %%% "phoenix-scala-native" % "0.1-SNAPSHOT"
+    )
+  } else Seq(
+    resolvers += "WPILib-Maven" at "http://team846.github.io/wpilib-maven",
+
+    libraryDependencies += "org.opencv" % "opencv-java" % "3.1.0",
```

Comment:
You should only depend on what you need. So the only dependency should be ntcore.